var/spack/repos/builtin/packages/libxpm/package.py

```diff
@@ -27,5 +27,7 @@ class Libxpm(AutotoolsPackage):
     depends_on('util-macros', type='build')
 
     def setup_environment(self, spack_env, run_env):
-        spack_env.append_flags('LDFLAGS', '-L{0} -lintl'.format(
-            self.spec['gettext'].prefix.lib))
+        # In case we're using gettext from the system.
```

Comment:
Can you change this comment to something like:

> If libxpm is installed as an external package, gettext won't be available in the spec. See https://github.com/spack/spack/pull/10881 for details.

Reply:
Done, see https://github.com/spack/spack/pull/10881/commits/8867d8048a05d1a30af60f0017e04612402e3c38.